src/Microsoft.Health.Dicom.Core/Features/CustomTag/CustomTagEntry.cs

```diff
@@ -35,23 +43,23 @@ public CustomTagEntry(CustomTagStoreEntry storedEntry)
         /// Path of this tag. Normally it's composed of groupid and elementid.
         /// E.g: 00100020 is path of patient id.
         /// </summary>
-        public string Path { get; private set; }
```

Comment:
sorry, not get you. with "private set", Asp.net core is not able to set the value, hence fail out.

Reply:
I see that you deleted the setter, and without it no code would set it via reflection (like presumably some automatically deserializer, like those in ASP.NET Core or JSON.NET). Are you keeping these values, or is the serializer using the ctor instead?

Reply:
eh...my intention is to keep both getter and setter, so that serializer  works.

Not  sure if there is a way for serializer to use the constructor with parameters.